docsite/docs/Docs/python-package.md

```diff
@@ -28,12 +28,6 @@ tickets = faldbt.ref('stg_zendesk_ticket_data')
 # pandas.DataFrame
 ```
 
-##  Interacting with dbt from a Notebook
```

Comment:
Should we delete this entirely or link to the new doc?

Reply:
I think we can delete it as it's not so relevant given `init_fal`.